### PR TITLE
【Hackathon 6th Fundable Projects 3 No.290】fluid operator pyramid_hash

### PR DESCRIPTION
**Pull request by co63oc**
### PR Category

Others

### PR Types

Others

### Description

迁移 pyramid_hash

### CODE REVIEW

**Review by zyfncg — DISMISSED** [review]

**Comment by luotao1** [comment]
需要解决下冲突

File: paddle/phi/kernels/cpu/pyramid_hash_grad_kernel.cc [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <xxhash.h>
+
+#include <algorithm>
+#include <cmath>
+
+#include "paddle/phi/core/compat/convert_utils.h"
```
Comment (zyfncg) [inline paddle/phi/kernels/cpu/pyramid_hash_grad_kernel.cc]: convert_utils.h的引用是否可以去掉？

File: paddle/phi/kernels/cpu/pyramid_hash_kernel.cc [hunk 2]
```diff
@@ -0,0 +1,270 @@
...
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <xxhash.h>
+#include <algorithm>
+#include <cmath>
+#include "paddle/phi/backends/cpu/cpu_context.h"
+#include "paddle/phi/core/compat/convert_utils.h"
```
Comment (zyfncg) [inline paddle/phi/kernels/cpu/pyramid_hash_kernel.cc]: 同上

Comment (co63oc) [reply paddle/phi/kernels/cpu/pyramid_hash_kernel.cc]: done

File: paddle/phi/kernels/cpu/pyramid_hash_grad_kernel.cc (hunk 1 above)
Comment (co63oc) [reply paddle/phi/kernels/cpu/pyramid_hash_grad_kernel.cc]: done

**Review by zyfncg — APPROVED** [review]

**Pull request merged**